### PR TITLE
Adding more concept tags to graph theory

### PR DESCRIPTION
**Pull request by EgbertRijke**

### CODE REVIEW

File: src/graph-theory/closed-walks-undirected-graphs.lagda.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ open import graph-theory.undirected-graphs
 ## Idea
 
 A
-{{#concept "closed walk" Agda=closed-walk-Undirected-Graph WDID=Q245595 WD="Cycle"}}
+{{#concept "closed walk" Agda=closed-walk-Undirected-Graph WDID=Q245595 WD="cycle"}}
```
[the remark this answers is not in the document]
Comment (EgbertRijke) [reply src/graph-theory/closed-walks-undirected-graphs.lagda.md:25]: Ah, that’s true. My intention was to adopt the convention of wikidata inside the WD field. Perhaps this was not a good idea.

[the remark this answers is not in the document]
Comment (EgbertRijke) [reply src/graph-theory/closed-walks-undirected-graphs.lagda.md]: I converted back to sentence casing.

File: src/graph-theory/cycles-undirected-graphs.lagda.md [hunk 2]
```diff
@@ -21,8 +21,10 @@ open import graph-theory.undirected-graphs
 
 ## Idea
 
-A **cycle** in an [undirected graph](graph-theory.undirected-graphs.md) `G`
-consists of a [`k`-gon](graph-theory.polygons.md) `H` equipped with an
+A
+{{#concept "cycle" Agda= Disambiguation="undirected graph" WD="Cycle" WDID=Q245595}}
```
Comment (VojtechStep) [inline src/graph-theory/cycles-undirected-graphs.lagda.md:25]: I believe the empty `Agda=` field will be properly ignored, but I'm not sure we want to have that?

Comment (EgbertRijke) [reply src/graph-theory/cycles-undirected-graphs.lagda.md:25]: No, that was an oversight on my part. It should probably raise an error:)

Comment (EgbertRijke) [reply src/graph-theory/cycles-undirected-graphs.lagda.md]: fixed

**Pull request merged**